components/brave_shields/resources/panel/style.ts

```diff
@@ -4,5 +4,5 @@ export const PanelWrapper = styled.div`
   width: 388px;
   min-height: 290px;
   position: relative;
-  overflow: hidden;
+  overflow: auto;
```

Comment:
This may fix by including a scrollbar, but it seems like a band-aid that maybe the design team wants to address if it's common. Although it will only affect advanced so maybe it's ok. Either way please can you attach a screenshot of what this visual change looks like as we can't easily see it in storybook.

Reply:
This has been significantly fixed.

<img width="472" alt="image" src="https://user-images.githubusercontent.com/8665427/168722728-ee0b6b56-9c94-48d1-a986-dee77f1b96e6.png">